pkg/kubelet/kubelet.go

```diff
@@ -2838,7 +2838,7 @@ func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
 }
 
 func extractBandwidthResources(pod *api.Pod) (ingress, egress *resource.Quantity, err error) {
-	str, found := pod.Annotations["kubernetes.io/ingress-bandwidth"]
+	str, found := pod.Annotations["net.beta.kubernetes.io/ingress-bandwidth"]
```

Comment:
That was my feeling too, so I was lazy.  I am fine with alpha, or can do
the compat dance...
On Oct 9, 2015 5:48 PM, "Derek Carr" <notifications@github.com> wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/15414#discussion_r41690399>
> :
>
> > @@ -2838,7 +2838,7 @@ func validateBandwidthIsReasonable(rsrc *resource.Quantity) error {
> >  }
> >
> >  func extractBandwidthResources(pod *api.Pod) (ingress, egress *resource.Quantity, err error) {
> > -	str, found := pod.Annotations["kubernetes.io/ingress-bandwidth"]
> > +	str, found := pod.Annotations["net.beta.kubernetes.io/ingress-bandwidth"]
>
> fwiw, i just started evaluating this last week, i cannot imagine there is
> wide use as it didn't get much in the way of documentation to my knowledge
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15414/files#r41690399>.
>